### PR TITLE
Add new type of user > Admin, and records validation for users

### PR DESCRIPTION
**Pull request by merykul**
- Admin user:
1. As an admin I want to have access to all  records, to be able to retrieve all data;
2. As an admin I want to be able to create, edit, delete user;
3. As an Admin I want to be able to assign data from records table, to existing or new user before user deletion, so data can be saved;

- User
1. Add records per user validation

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    if current_user.role == 'admin'
+      @users = User.where(:role => 'user')
+      erb :'user/manage'
+    else
+      erb :'errors/users_profiles_access_error'
+    end
+  end
+
+  get '/create_user' do
+    if current_user.role == 'admin'
```
Comment (olichka12) [inline app/controllers/users_controller.rb]: Could be used ternary operator

Comment (merykul) [reply app/controllers/users_controller.rb]: Adjusted ✅

File: app/controllers/records_controller.rb [hunk 2]
```diff
@@ -1,112 +1,79 @@
...
-      end
-    else
-      redirect to '/start'
-    end
+    redirect_if_not_logged_in
+    city_filter = params[:city]
+    user_id = current_user.id
+    @records = current_user.role == 'admin' ? Records.all : Records.where(:user_id => user_id)
+
+    @output = if city_filter && !city_filter.strip.empty?
```
Comment (olichka12) [inline app/controllers/records_controller.rb]: Could be in one row by using ternary

File: app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -45,34 +52,21 @@ class SessionsController < ApplicationController
...
-    else
-      @error_messages = @user.errors.full_messages
-      erb :'user/sign_up_page'
-    end
+    create_user('/homepage', { username: params[:username],
+                               password: params[:password],
+                               first_name: params[:first_name],
+                               second_name: params[:second_name],
+                               password_confirmation: params[:password_confirmation],
+                               password_status: 'permanent' }, :'user/sign_up_page')
```
Comment (olichka12) [inline app/controllers/sessions_controller.rb]: That to be more readable 
could be in this way
opts = {username: params[],......}
create_user('page', opts, :'page')

**Review by olichka12 — APPROVED** [review]
Great work 👍

**Pull request merged**